test/elastic-adapter.spec.js

```diff
@@ -247,6 +265,44 @@ describe('elastic source', function() {
         });
     });
 
+    describe('write edge cases', function() {
+        after(function() {
+            return test_utils.clear_data();
+        });
+
+        it('writes a nested object', function() {
+            var nested_object = {
+                time: new Date().toISOString(),
+                nest: {nested_key: 'nest'},
+                name: 'nest_haver'
+            };
+            return test_utils.write([nested_object])
+                .then(function(result) {
+                    expect(result.errors).deep.equal([]);
+                    return retry(function() {
+                        return test_utils.search()
+                            .then(function(result) {
+                                var hits = _.pluck(result.hits.hits, '_source');
+                                var imported = _.findWhere(hits, {name: 'nest_haver'});
+
+                                expect(imported).exist; // jshint ignore:line
+                            });
+                    });
+                });
+        });
+
+        it('fails to write a point with an _id field', function() {
```

Comment:
Shouldn't this test belong with the other `idField` PR? I don't see what it has to do with the changes here.